### ISSUE TITLE
`xvalues.cpp`: Remove unused non-constant "constants" in vNext

### ISSUE THREAD

**Issue by AlexGuteniev** [microsoft/STL#3622]
This machinery is DLL-exported but unused:

https://github.com/microsoft/STL/blob/cbe2ee99caf122555a305d18f69e57c75b3fe5ec/stl/src/xvalues.cpp#L19-L38

**vNext note:** Resolving this issue will require breaking binary compatibility. We won't be able to accept pull requests for this issue until the vNext branch is available. See https://github.com/microsoft/STL/issues/169 for more information.

**Comment by frederick-vs-ja** [comment]
Some internal functions are no longer used and not DLL-exported. Can we drop them right now?

**Comment by StephanTLavavej** [comment]
@frederick-vs-ja Yes, but you have to be very careful about searching for usage. Some of them are cleverly macroized for float/double/long double variants, so the identifiers appear to be unused at first.

**Comment by StephanTLavavej** [comment]
#5836 radically simplified this, so the remaining work in vNext will be trivial.
